client/devdocs/design/search-collection.jsx

```diff
@@ -35,6 +35,14 @@ const shouldShowInstance = ( example, filter, component ) => {
 };
 
 const getReadmeFilePath = ( section, example ) => {
+	if ( ! example.props.readmeFilePath ) {
+		return null;
+	}
+
+	if ( example.props.readmeFilePath.charAt( 0 ) === '/' ) {
+		return `${ example.props.readmeFilePath }/README.md`;
```

Comment:
Should we do that only if the path doesn't end with `README.md`? I think it make things even more future-proof.

Reply:
I'm going to make the code a bit smarter to account for this kind of case as well 👍